docs/actions.md

```diff
@@ -939,12 +945,13 @@ You can list all the actions that you have created using:
 wsk action list
 ```
 
-As you write more actions, this list gets longer and it can be helpful to group related actions into [packages](./packages.md). To filter your list of actions to just the those within a specific pacakge, you can use: 
+By default, listed actions will be sorted alphabetically, first by namespaces, then [packages](./packages.md), and finally by action name. To filter your list of actions to just those within a specific package, you can use:
```

Comment:
@underwoodb-sd-ibm - by saying "By default", it seems to imply that there are non-default sorting options?  May need to document those....?